src/app/routes/liveRadio/getInitialData/index.test.js

```diff
@@ -1,14 +1,13 @@
 import getInitialData from '.';
 import * as fetchPageData from '../../utils/fetchPageData';
 import liveRadioJson from '#data/korean/bbc_korean_radio/liveradio.json';
+import { MEDIA_PAGE as pageType } from '#app/routes/utils/pageTypes';
```

Comment:
Can we go a step further and remove `as pageType` and use `MEDIA_PAGE` directly?